lib/haml_coffee_assets/action_view/resolver.rb

```diff
@@ -8,7 +8,7 @@ module ActionView
     # in it would normally be a fallback for all formats.
     #
     class Resolver < ::ActionView::FileSystemResolver
-      def find_templates(name, prefix, partial, details)
+      def find_templates(name, prefix, partial, details, outside_app_allowed = false)
```

Comment:
@aprescott 

>> This would break versions of Rails that still use 4 arguments, right?

Upon further consideration, in fact I think you are wrong, it *won't* break any calling code that still use 4 arguments, since by design Ruby will treat the 5th argument as optional.

I don't know why that didn't occur to me earlier this morning